### ISSUE TITLE
Implement ComponentSelectorTableView

### ISSUE THREAD

**Issue by jsbean** [jsbean/DNM#132]

**Comment by jsbean** [comment]
Implement Leader TableViewCell

Create 
- ComponentSelectorTableViewCellPerformer
- ComponentSelectorTableViewCellComponent

Different graphical, and behavioral properties for each.

**Comment by jsbean** [comment]
But actually has to be a set of ComponentSelectorTableViews (one for each performer).

**Comment by jsbean** [comment]
Distant incarnation could be a hierarchical menu with the data:

```JSON
{
    "performer1": [
        "identifier",
        "dynamics",
        "articulations",
        "other stuff",
        "instruments": [
            "instrument1":
                    "instrument1_id"
                    "graph1_id"
                ]
            ]
        ]
    ]
}

maybe?
```

**Comment by jsbean** [comment]
Closed with a75e8476393020b96d41c5ae2fdaad9c2843f665
